File: app/src/main/java/com/example/appbank/Data/ILoginService.java
package com.example.appbank.Data;

import com.example.appbank.Domain.Model.UserAccount;

import okhttp3.RequestBody;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface ILoginService {
    @POST("login")
    Call<UserAccount> Login(@Body RequestBody object);
}

File: BankApp/app/src/main/java/br/com/rphmelo/bankapp/login/api/ILoginService.kt
package br.com.rphmelo.bankapp.login.api

import br.com.rphmelo.bankapp.login.domain.models.LoginRequest
import br.com.rphmelo.bankapp.login.domain.models.LoginResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface ILoginService {

    @POST("login")
    fun login(@Body request: LoginRequest): Call<LoginResponse>
}